andela/AndelaEatsUI
# #164197344 Improve test coverage

#### What does this PR do?

- Improves the test coverage

#### Description of Task to be completed?

- add more tests to boost the test coverage
#### How should this be manually tested?

- run the tests with coverage, you should be able to see an improvement on the test coverage

#### What are the relevant pivotal tracker stories?

- [#164197344](https://www.pivotaltracker.com/story/show/164197344)
